pkg/server/pubsub_test.go

```diff
@@ -106,3 +114,15 @@ func newTestPubSubGrpcConn(ctx context.Context, t *testing.T, opts ...pstest.Ser
 
 	return conn
 }
+
+func createTestMessangerTopic(ctx context.Context, projectID, topicID string, opts ...option.ClientOption) (*pubsub.Topic, error) {
```

Comment:
Errors should be handled directly in the test helper, see https://google.github.io/styleguide/go/best-practices#test-helper-error-handling
Test helper should be named `testXxx` https://github.com/abcxyz/readability/wiki/Go-Best-Practices#test-helper